webui/src/lib/components/repository/changes.jsx

```diff
@@ -139,31 +132,26 @@ export const TreeEntryRow = ({ entry, showActions, relativeTo="", leaf=false, ex
     );
 };
 
-export const ChangeEntryRow = ({ entry, showActions, relativeTo="", onNavigate, onRevert }) => {
-    let rowClass = 'change-entry-row ' + diffType(entry);
-    let pathText = extractPathText(entry, relativeTo);
-    let diffIndicator = diffIndicatorIcon(entry);
-    let actions = entryActions(showActions, entry, onRevert);
-
+export const TreeEntryPaginator = ({ path, setAfterUpdated, nextPage, depth=0, loading=false }) => {
+    let pathText = "Load more results ...";
```

Comment:
nit; you may want to call this "pathSectionText"

Reply:
Done.